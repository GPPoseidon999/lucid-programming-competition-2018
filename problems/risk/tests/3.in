100
250 200
250 201
250 202
250 203
250 204
250 205
250 206
250 207
250 208
250 209
250 210
250 211
250 212
250 213
250 214
250 215
250 216
250 217
250 218
250 219
250 220
250 221
250 222
250 223
250 224
250 225
250 226
250 227
250 228
250 229
250 230
250 231
250 232
250 233
250 234
250 235
250 236
250 237
250 238
250 239
250 240
250 241
250 242
250 243
250 244
250 245
250 246
250 247
250 248
250 249
250 250
250 251
250 252
250 253
250 254
250 255
250 256
250 257
250 258
250 259
250 260
250 261
250 262
250 263
250 264
250 265
250 266
250 267
250 268
250 269
250 270
250 271
250 272
250 273
250 274
250 275
250 276
250 277
250 278
250 279
250 280
250 281
250 282
250 283
250 284
250 285
250 286
250 287
250 288
250 289
250 290
250 291
250 292
250 293
250 294
250 295
250 296
250 297
250 298
250 299
