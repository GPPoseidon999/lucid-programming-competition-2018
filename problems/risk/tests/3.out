0.9997199
0.9996710
0.9996197
0.9995552
0.9994877
0.9994032
0.9993152
0.9992055
0.9990918
0.9989505
0.9988045
0.9986240
0.9984383
0.9982094
0.9979747
0.9976867
0.9973926
0.9970329
0.9966670
0.9962212
0.9957693
0.9952207
0.9946667
0.9939967
0.9933224
0.9925097
0.9916948
0.9907162
0.9897383
0.9885679
0.9874025
0.9860125
0.9846331
0.9829934
0.9813718
0.9794504
0.9775566
0.9753200
0.9731230
0.9705365
0.9680041
0.9650323
0.9621320
0.9587392
0.9554385
0.9515894
0.9478565
0.9435168
0.9393211
0.9344582
0.9297713
0.9243551
0.9191508
0.9131545
0.9074100
0.9008108
0.8945074
0.8872870
0.8804104
0.8725560
0.8650972
0.8566018
0.8485575
0.8394206
0.8307933
0.8210214
0.8118203
0.8014266
0.7916673
0.7806727
0.7703774
0.7588098
0.7480072
0.7359016
0.7246267
0.7120248
0.7003187
0.6872685
0.6751777
0.6617329
0.6493087
0.6355280
0.6228260
0.6087721
0.5958512
0.5815903
0.5685119
0.5541123
0.5409394
0.5264707
0.5132671
0.4987993
0.4856285
0.4712307
0.4581551
0.4438946
0.4309746
0.4169163
0.4042094
0.3904147
